lib/features/pokemon_overview/pokemon_tile.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:pokedex/utils/strings.dart' as str;
+
+class PokemonTile extends StatefulWidget {
```

Comment:
- do you really need to make this stateful? 🤔 
- kindly rename this to PokemonTileItem 🙂 
- is this something to be reused for favorites as well right? If so, kindly move this to your general widget directory 🙂